src/libs/actions/PersonalDetails.js

```diff
@@ -166,13 +166,16 @@ function getFromReportParticipants(reports) {
             // The personalDetails of the participants contain their avatar images. Here we'll go over each
             // report and based on the participants we'll link up their avatars to report icons.
             _.each(reports, (report) => {
-                if (report.participants.length === 1) {
-                    const dmParticipant = report.participants[0];
-                    let icon = lodashGet(details, [dmParticipant, 'avatar'], '');
-                    if (!icon) {
-                        icon = getDefaultAvatar(dmParticipant);
-                    }
-                    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`, {icon});
+                if (report.participants.length > 0) {
+                    const avatars = report.participants
+                        .map(dmParticipant => ({
+                            firstName: lodashGet(details, [dmParticipant, 'firstName'], ''),
+                            avatar: lodashGet(details, [dmParticipant, 'avatar'], getDefaultAvatar(dmParticipant)),
+                        }))
+                        .sort((first, second) => first.firstName - second.firstName)
+                        .map(item => item.avatar);
+
+                    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${report.reportID}`, {icon: avatars});
```

Comment:
This would make more sense if it was changed to `icons`. It's strange that `icon` can be one ore more icons.